src/css/button.scss

```diff
@@ -5,3 +5,42 @@ a.button {
 a.button-primary {
   @extend .btn-primary;
 }
+
+.btn {
```

Comment:
@maymax777 , class `btn` has already been defined in Bootstrap. You can define custom styles as `a.button` on the top.